src/ClientResources/TagsSelection.js

```diff
@@ -29,7 +29,9 @@ function (
                 allowDuplicates: this.allowDuplicates,
                 caseSensitive: this.caseSensitive,
                 readOnly: this.readOnly,
+                singleFieldDelimiter : this.singleFieldDelimiter,
                 tagLimit: this.tagLimit !== -1 ? this.tagLimit : null,
+                convertFromOldDelimeter: this.convertFromOldDelimeter,
```

Comment:
Do you use this property somewhere? (convertFromOldDelimeter)

Reply:
no.. i was... but I removed it because of the logic.. thanks for pointing this...